website/content/docs/other-specifications/quota.mdx

```diff
@@ -0,0 +1,88 @@
+---
+layout: docs
+page_title: Nomad Quota Specification
```

Comment:
I noticed (some) other pages include Nomad in the title and assumed it might be for SEO? but every page already has `| Nomad | Hashi...` in the title anyway, so now I'm not so sure!  I'm happy to drop it :)